docs/source/data/data_catalog.md

```diff
@@ -1,31 +1,34 @@
-# The Data Catalog
+# Introduction to the Data Catalog
 
-This section introduces `catalog.yml`, the project-shareable Data Catalog. The file is located in `conf/base` and is a registry of all data sources available for use by a project; it manages loading and saving of data.
+## The basics of `catalog.yml`
+A separate page of [Data Catalog YAML examples](./data_catalog_yaml_examples.md)  gives further examples of how to work with `catalog.yml`, but here we revisit the [basic `catalog.yml` introduced by the spaceflights tutorial](../tutorial/set_up_data.md).
 
-All supported data connectors are available in [`kedro-datasets`](/kedro_datasets).
+The example below registers two `csv` datasets, and an `xlsx` dataset. The minimum details needed to load and save a file within a local file system are the key, which is name of the dataset, the type of data to indicate the dataset to use (`type`) and the file's location (`filepath`).
```

Comment:
I'm wondering if we could make the beginning of this page a bit more self-contained. It's probably going to be among the most visited ones, and seeing these two links in the first sentence overwhelmed me a bit.

I see the parent page has the sentence

> The catalog is stored in a YAML file (catalog.yml) that maps the names of node inputs and outputs as keys in the DataCatalog class.

But maybe a bit of an intro here before explaining the example below would be good too?

Reply:
Yeah, it's a bit of a tricky one because I put this introductory information in the `index.md` page but retained the detail of catalog on this page separately. In part because I'm struggling to work out how to put content onto `index.md` if it has subheads, because they then show up in the ToC and that messes up the nav.

But also because I think it's wise to keep the meat of the docs on already-popular `data_catalog.md`.

Maybe it's OK to reproduce the intro section. Let me drop it back in and see what you think.